Lib/idlelib/idle_test/test_iomenu.py

```diff
@@ -36,6 +39,14 @@ def tearDownClass(cls):
         cls.root.destroy()
         del cls.root
 
+    def _create_tempfile(self, content: str) -> str:
+        fd, filename = tempfile.mkstemp(suffix='.py')
+        os.close(fd)
+        self.addCleanup(os.unlink, filename)
+        with builtins.open(filename, 'w', encoding='utf-8') as f:
+            f.write(content)
+        return filename
```

Comment:
```suggestion
        filename = tempfile.mktemp()
        self.addCleanup(os_helper.unlink, filename)
        with open(filename, "w") as f:
            f.write(content)
        return filename
```

We don't care about secure temporary file for a test.